hedera-mirror-rest/__tests__/specs/blocks-01-no-args.spec.json

```
@@ -0,0 +1,124 @@
+{
+  "description": "Get blocks with no arguments",
+  "setup": {
+    "recordFiles": [
+      {
+        "index": 16,
+        "count": 3,
+        "hapi_version_major": "0",
+        "hapi_version_minor": "22",
+        "hapi_version_patch": "3",
+        "name": "2022-04-27T12_09_24.499938763Z.rcd",
+        "prev_hash": "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+        "consensus_start": 1676540001234390000,
+        "consensus_end": 1676540001234490000,
+        "hash": "fbd921184e229e2051280d827ba3b31599117af7eafba65dc0e5a998b70c48c0492bf793a150769b1b4fb2c9b7cb4c1c"
+      },
+      {
+        "index": 17,
+        "count": 5,
+        "hapi_version_major": "0",
+        "hapi_version_minor": "22",
+        "hapi_version_patch": "3",
+        "name": "2022-04-27T12_24_30.768994443Z.rcd",
+        "prev_hash": "fbd921184e229e2051280d827ba3b31599117af7eafba65dc0e5a998b70c48c0492bf793a150769b1b4fb2c9b7cb4c1c",
+        "consensus_start": 1676540001234500000,
+        "consensus_end": 1676540001234600000,
+        "hash": "b0162e8a244dc05fbd6f321445b14dddf0e94b00eb169b58ff77b1b5206c12782457f7f1a2ae8cea890f378542ac7216"
+      },
+      {
+        "index": 18,
+        "count": 4,
+        "hapi_version_major": "0",
+        "hapi_version_minor": "22",
+        "hapi_version_patch": "3",
+        "name": "2022-04-27T12_49_14.923546113Z.rcd",
+        "prev_hash": "b0162e8a244dc05fbd6f321445b14dddf0e94b00eb169b58ff77b1b5206c12782457f7f1a2ae8cea890f378542ac7216",
+        "consensus_start": 1676540001234610000,
+        "consensus_end": 1676540001234700000,
```

wrap the timestamps as string since they are larger than nodejs MAX_SAFE_INTEGER, may cause issue
```suggestion
        "consensus_start": "1676540001234610000",
        "consensus_end": "1676540001234700000",
```